src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,20 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.UserNotFoundException;
+
 public class UserService {
 
     public int getUserScore(String[] records, String email) {
-        return 0;
+        int userScope = -1;
+        for (String user : records) {
+            if (user.matches(email + ".*") && !email.matches(".*[0-9]")) {
+                String[] userRecords = user.split(":");
+                userScope = Integer.parseInt(userRecords[1]);
+            }
+        }
+        if (userScope == -1) {
+            throw new UserNotFoundException("User with given email doesn't exist");
```

Comment:
throw exception if loop came to an end. This way you won't need userScore variable

Reply:
done